settings.gradle

```diff
@@ -8,6 +8,7 @@ include 'tests:example-success'
 include 'tests:example-success-java17'
 include 'tests:example-success-java21'
 include 'tests:example-syntax-error'
+include 'tests:example-junit4-nested'
```

Comment:
Done! I was unsure about either `junit4-nested` (because the tests are in nested classes) or `junit4-enclosed` (because of the `Enclosed` test runner), but I completely agree.